advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup PostgreSQL Database Server
+1. From the navigation pane, go to Protect > Databases.
+2. Click Add server.
+3. Select the database type which in this case is PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd1.png">
+</p>
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
+6. From the Plan list, select the server plan to use for the database server.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd2.png">
+</p>
+
+7. Once server is added, then a Job will run to install client on the Server. Below screenshots display the entire process were a Job is created to install Client on the Server and then Dowloaded it and installed it.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd3.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd4.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd5.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd6.png">
+</p>
+
+### Setup EDB Postgres Advanced Server (EPAS) Instance
+1. From the navigation pane, go to Protect > Databases > DB Instances.
+2. Click Add instance, and then select PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/CreateInstance1.png">
+</p>
+
+3. From the Server name list, select a server where you want to create the new instance.
+4. In the Instance Name box, type the EDB Postgres Advanced Server (EPAS)  instance name.
+5. From the Plan list, select a server plan.
+6. Under Connection details, enter the following information
+   - In the Database user box, type the user name to access the EDB Postgres Advanced Server (EPAS) instance.
+   - In the Password box, type the EDB Postgres Advanced Server (EPAS) user account password.
+   - In the Port box, type the port to open the communication between EDB Postgres Advanced Server (EPAS) Server and the clients.
+   - In the Maintenance DB box, type the name of a system database which is used as a maintenance database.
+   - In PostgreSQL section enter paths for Binary Directory, Lib Directory and Archive Log Directory.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/CreateInstance2.png">
+</p>
+
+7. Once the Instance is created then its information will be disoplayed as shown by below screenshots.
```

Comment:
7. Your database instance to backup is now created.  You can now view it's configureation.